backend/src/test/java/de/ffmjava/capstone/backend/horses/HorseIntegrationTest.java

```diff
@@ -28,4 +30,49 @@ void getAllHorses() throws Exception {
                 .andExpect(status().isOk())
                 .andExpect(content().json("[]"));
     }
+
+    @Test
+    @DirtiesContext
+    @WithMockUser(roles = "Basic")
+    void addNewHorse_AndExpectSuccessMessage_201() throws Exception {
+        String jsonString =
+                """
+                            {
+                              "name": "Hansi",
+                              "owner": "Peter Pan",
+                              "consumption": []
+                            }
+                        """;
+        mockMvc.perform(post("/horses/")
+                        .contentType(MediaType.APPLICATION_JSON)
+                        .content(jsonString)
+                )
+                .andExpect(status().is(201))
+                .andExpect(jsonPath("$.id").isNotEmpty())
+                .andExpect(jsonPath("$.name").isNotEmpty())
+                .andExpect(jsonPath("$.owner").isNotEmpty())
+                .andExpect(jsonPath("$.consumption").isEmpty());
```

Comment:
I would add more accurate  test   ex: name is "Hansi" etc...
maybe assertEquals(jsonString, content from response) ?!